app/controllers/standup_meeting_groups/joins_controller.rb

```diff
@@ -7,12 +8,8 @@ def create
 
     authorize @standup_meeting_group_user, policy_class: StandupMeetingGroup::JoinPolicy
 
-    @standup_meeting_group_user.save!
```

Comment:
According to the [API docs](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save-21), the only two errors that could be raised by `#save!` are:
- `ActiveRecord::RecordInvalid` due to validation failure
- `ActiveRecord::RecordNotSaved` due to an associated `before_*` callback throwing a `:abort`

If there are existing callbacks that could throw `:abort`, perhaps they could be handled at a higher level, outside the scope of an individual controller since `StandupMeetingGroups::JoinsController` by itself doesn’t seem to be capable of triggering such a scenario.